sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,17 +1,13 @@
 ## Release History
 
-### 3.27.0-beta.1 (Unreleased)
+### 3.27.0 (2022-09-15)
 
 #### Features Added
 * Exposed `maxBufferedItemCount` feature from `CosmosQueryRequestOptions` through `application.properties` flag - See [PR 30921](https://github.com/Azure/azure-sdk-for-java/pull/30921)
 
-#### Breaking Changes
-
 #### Bugs Fixed
 * Fixing pagination bug when performing a cross-partition query to fill every page and fix the total page count reporting. - See [PR 30694](https://github.com/Azure/azure-sdk-for-java/pull/30694)
 
```

Comment:
We are missing other changes section for spring-data-cosmos. Please add that. (see the previous release notes for reference)

Reply:
done!